specification/widget/resource-manager/Microsoft.Widget/WidgetManagement/readme.md

```diff
@@ -0,0 +1,55 @@
+# containerstorage
+
+> see https://aka.ms/autorest
+> This is the AutoRest configuration file for Contoso.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+> To see additional help and options, run:
+
+> `autorest --help`
+> For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the containerstorage.
+
+```yaml
+openapi-type: arm
+openapi-subtype: rpaas
+tag: package-2021-11-01
+```
+### Suppressions
+
+```yaml
+suppressions:
+  - code: PathContainsResourceType
+  - code: PathResourceProviderMatchNamespace
```

Comment:
@markcowl: Does it make sense we need to suppress these? Or does this indicate a likely bug in my spec?